Data Model/mainProperties.js

```diff
@@ -110,6 +116,359 @@ export const data =
             }
             //https://dbpedia.org/ontology/denomination
         },
+        {
+            id:  'P12',
```

Comment:
in my opinion it should be null now - because this property is not created in wiki yet